kots-exporter/kots-exporter.sh

```diff
@@ -485,8 +467,8 @@ elif [[ "$func" == "all" ]]; then
     if [[ $annotate == 1 ]]; then
         annotation_k8s_resource
     fi
-    execute_flyway_migration
     rm_kots_annot_label_resources
+    execute_flyway_migration
```

Comment:
By running `execute_flyway_migration` at last, in case of `execute_flyway_migration` fail (which terminates the script as well), allow us to only re-run the  `execute_flyway_migration`
Currently, we have to run 2 function  (`execute_flyway_migration` and `rm_kots_annot_label_resources`)